### PR TITLE
Disabling next/previous buttons at course beginning and end

### PR DESCRIPTION
**Pull request by davidjoy**
Task: https://openedx.atlassian.net/browse/TNL-7111

At the end of a course, the “next” button at the bottom of the page is replaced with a friendly message. 🤗 

This PR also alphabetizes some props for SequenceNavigation and Sequence, as I was adding two new ones - isFirst and isLast.

### CODE REVIEW

File: src/courseware/course/SequenceContainer.jsx [hunk 1]
```diff
@@ -68,7 +71,9 @@ function SequenceContainer(props) {
   }, [isTimeLimited]);
 
   const isLoading = !loaded || !unitId || isTimeLimited;
-
+  const isFirst = sequenceIds.indexOf(sequenceId) === 0 && unitIds.indexOf(unitId) === 0;
```
Comment (abutterworth) [inline src/courseware/course/SequenceContainer.jsx]: `isFirstUnit`?

File: src/courseware/course/SequenceContainer.jsx [hunk 2]
```diff
@@ -30,10 +31,12 @@ function SequenceContainer(props) {
     position,
     items,
     lmsWebUrl,
+    models,
   } = props;
   const loaded = fetchState === 'loaded';
 
   const unitIds = useMemo(() => items.map(({ id }) => id), [items]);
+  const sequenceIds = useMemo(() => createSequenceIdList(models, courseId), [models, courseId]);
```
Comment (abutterworth) [inline src/courseware/course/SequenceContainer.jsx:39]: What do you think about doing this work in mapStateToProps?

File: src/courseware/course/SequenceContainer.jsx (hunk 1 above)
Comment (davidjoy) [reply src/courseware/course/SequenceContainer.jsx]: Hm - eh, sure. :) Will change.

**Review by abutterworth — APPROVED** [review]

File: src/courseware/course/SequenceContainer.jsx (hunk 2 above)
Comment (davidjoy) [reply src/courseware/course/SequenceContainer.jsx:39]: Eh, we haven't been doing much derived state in there, so I've been avoiding it.  We also do something very similar in Course.jsx, and it's not connected to redux, so it doesn't have a similar opportunity.  

We were talking about whether or not we wanted to refactor anything for readability - totally open to tweaking our approach a bit when we do that.  For now I'm not sure which is really better.

**Pull request merged**